### PR TITLE
fix: update to right vi 2.0 colours

### PR DESCRIPTION
**Pull request by zuri17**
Change the vi 1.0 colours for the current vi 2.0 ones.

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,26 +1,26 @@
 /* ------ VI colors, done by Philip ------------- */
```
Comment (WikiRik) [inline src/styles/main.scss:1]: Maybe remove this 'done by Philip'?

**Comment by WikiRik** [comment]
Didn't realize we still worked with old colours, thanks for updating this!

**Comment by WikiRik** [comment]
> there are other tabs to fix: can you generally use only spaces instead of tabs?

And on this; can we check this with a linter?

**Comment by zuri17** [comment]
@WikiRik I can remove the Philip comment indeed, didn't even realise
@linuxbandit I kept the tabs because it's what I found and didn't know the standard. As a standard declaration I can leave just one space, but indeed it would be useful to have a check on that as Rik said

File: src/styles/main.scss (hunk 1 above)
Comment (serge1peshcoff) [reply src/styles/main.scss]: we can add Zuri here and leave Philip here, as he was the one doing it from the start

Comment (serge1peshcoff) [reply src/styles/main.scss]: any solution is fine for me

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,26 +1,26 @@
-/* ------ VI colors, done by Philip ------------- */
+/* ------ VI 2.0 colourrs */
```
Comment (serge1peshcoff) [inline src/styles/main.scss:1]: *colours*

**Comment by zuri17** [comment]
Damn me

File: src/styles/main.scss (hunk 2 above)
Comment (zuri17) [reply src/styles/main.scss:1]: Damn me xD something else I should change?

Comment (serge1peshcoff) [reply src/styles/main.scss:1]: other than that seems fine

**Comment by serge1peshcoff** [comment]
@zuri17 can I merge this if all checks would pass?

**Comment by linuxbandit** [comment]
> @WikiRik I can remove the Philip comment indeed, didn't even realise
> @linuxbandit I kept the tabs because it's what I found and didn't know the standard. As a standard declaration I can leave just one space, but indeed it would be useful to have a check on that as Rik said

Find a plugin for your editor that automatically reads [.editorconfig](https://github.com/AEGEE/frontend/blob/30e5156fed8f7a3d44526abe00992c93b1c3ef7b/.editorconfig) files and transforms everything into the established standard

**Pull request merged**